benefits/templates/error-base.html

```diff
@@ -0,0 +1,26 @@
+{% extends "core/base.html" %}
+{% load i18n %}
+
+{% block page-title %}
+  {% block title %}
+  {% endblock title %}
+{% endblock page-title %}
```

Comment:
Is this necessary? Test.

Reply:
I wouldn't think so. I would think the inheriting error page e.g. `404.html` or whatever could just use the
`{% block page-title %}` directly, right?

Reply:
So this was not necessary, but I had to then go back to each `40X` error page and change `page-title` to `title.

This anti-pattern is also exists on `landing`/`agency-index` (which btw is confusing naming. `landing-base`/`agency-landing` might be less confusing.